tests/application/detect/test_pyav_frame_count_provider.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+import pytest
+
+from OTVision.detect.pyav_frame_count_provider import PyAVFrameCountProvider
```

Comment:
Test and class are not in the same module path. One is in `OTVision.detect` other in `OTVision.application.detect`